cmd/release/cmd/update.go

```diff
@@ -42,8 +44,41 @@ var updateK3sReferencesCmd = &cobra.Command{
 	},
 }
 
+var updateChartsCmd = &cobra.Command{
+	Use:   "charts [branch] [chart] [version]",
+	Short: "Update charts files locally, stage and commit the changes.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var branch, chart, version string
+
+		if len(args) < 3 {
+			return errors.New("expected at least three arguments: [branch] [chart] [version]")
+		}
+
+		branch = args[0]
+		chart = args[1]
+		version = args[2]
+
+		config := rootConfig.Charts
+		if config.Workspace == "" || config.ChartsForkURL == "" {
+			return errors.New("verify your config file, chart configuration not implemented correctly, you must insert workspace path and your forked repo url")
+		}
+
+		ctx := context.Background()
+		gh := repository.NewGithub(ctx, rootConfig.Auth.GithubToken)
+
+		output, err := charts.Update(ctx, gh, config, branch, chart, version)
+		if err != nil {
+			fmt.Println(output)
+			return err
+		}
+
+		return nil
```

Comment:
```suggestion
		output, err := charts.Update(ctx, gh, config, branch, chart, version)
		fmt.Println(output)
		return err
```

Reply:
Since `charts.Update` also returns an output on successful runs, maybe you'd wan't to also print even if err is nil.

Reply:
Agreed, changed